pkg/version/version_test.go

```diff
@@ -18,10 +18,11 @@ import "testing"
 
 func TestBuildInfo(t *testing.T) {
 	cases := []struct {
-		name     string
-		in       BuildInfo
-		want     string
-		longWant string
+		name        string
+		in          BuildInfo
+		want        string
+		longWant    string
+		longWantAlt string
```

Comment:
What is this new field, where does it come from, is there a doc ?

Reply:
Afaict the anonymous struct is only used within this function to compare the output of the function against a hard-coded string.  The string is hard-coded for go 1.10, so I made an alternate string that can be matched against to pass the test (in this case a hard-coded string with 1.9.3).  I suppose I could go a step further and have this alternate be a generated string that matches the go version.  I could just use this generated string without having an alternate but it might defeat the utility of the test (i.e., if the actual code and the test code are using the same string substitution mechanism then I don't think the test provides much value [while then also introducing a shadow implementation to keep consistent with the actual).

Reply:
I came up with an alternate implementation that's dynamic yet sufficiently different in implementation to retain some value of the test.  I meant to put it on a different branch for comparison but I fat-fingered the push.  I think I like the use of runtime.Version() better than what istio is using in the actual code (i.e., -X with bin/get_workspace_status), though I probably need guidance on how to make it look more like go code.  The var at the top looks too much like C, but if I stick it in the struct it gets a bit tough to read.